plugins/lookup/aws_secret.py

```diff
@@ -51,6 +69,12 @@
      password: "{{ lookup('aws_secret', 'DbSecret') }}"
      tags:
        Environment: staging
+
+ - name: skip if secret does not exist
+   debug: msg="{{ lookup('aws_secret', 'secret-not-exist', on_missing='skip')}}"
+
+ - name: skip if access to the secret is denied
+   debug: msg="{{ lookup('aws_secret', 'secret-denied', on_denied='skip')}}"
```

Comment:
```suggestion
 - name: warn if access to the secret is denied
   debug: msg="{{ lookup('aws_secret', 'secret-denied', on_denied='warn')}}"
```

It's nice to give an example of both new behaviours.